posedetection_msgs/package.xml

```diff
@@ -14,20 +14,24 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>roscpp</build_depend>
-  <build_depend>std_msgs</build_depend>
-  <build_depend>sensor_msgs</build_depend>
-  <build_depend>geometry_msgs</build_depend>
   <build_depend>cv_bridge</build_depend>
-  <build_depend>message_generation</build_depend>
+  <build_depend>geometry_msgs</build_depend>
   <build_depend>message_filters</build_depend>
+  <build_depend>message_generation</build_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>sensor_msgs</build_depend>
+  <build_depend>std_msgs</build_depend>
+  <build_depend>tf2_geometry_msgs</build_depend>
+  <build_depend>tf2_ros</build_depend>
 
-  <run_depend>roscpp</run_depend>
-  <run_depend>std_msgs</run_depend>
-  <run_depend>sensor_msgs</run_depend>
-  <run_depend>geometry_msgs</run_depend>
   <run_depend>cv_bridge</run_depend>
-  <run_depend>message_runtime</run_depend>
+  <run_depend>geometry_msgs</run_depend>
   <run_depend>message_filters</run_depend>
+  <run_depend>message_runtime</run_depend>
```

Comment:
please separate style change and your contribution, when I first see this patch, it seems you have increased a lot of dependencies and since the `jsk_common_msgs` is the root node of jsk repositories, it is not acceptable.